### PR TITLE
Log UTxO size directly from validation and update readme

### PR DESCRIPTION
**Pull request by ruhatch**

### CODE REVIEW

File: release.nix [hunk 1]
```diff
@@ -55,8 +55,8 @@ localLib.pkgs.lib.mapAttrsRecursiveCond
     jobs.nix-tools.tests.cardano-ledger.cardano-ledger-test.x86_64-linux
 
     # windows cross compilation targets
-    jobs.nix-tools.libs.x86_64-pc-mingw32-cardano-ledger.x86_64-linux
-    jobs.nix-tools.tests.x86_64-pc-mingw32-cardano-ledger.cardano-ledger-test.x86_64-linux
+    # jobs.nix-tools.libs.x86_64-pc-mingw32-cardano-ledger.x86_64-linux
```
Comment (nc6) [inline release.nix:58]: Why are these now commented? Related to this PR?

**Review by nc6 — APPROVED** [review]
LGTM

File: release.nix (hunk 1 above)
Comment (ruhatch) [reply release.nix:58]: They were failing on this PR because of a new `katip` dep, but I hope rebasing on @angerman's PR will fix it

**Pull request merged**